main.py

```diff
@@ -44,7 +44,7 @@ async def on_ready(self):
 
 bot.active_chat = []
 
-bot.ddae_version = "0.1.0"
+bot.ddae_version = "0.1.1"
```

Comment:
should be 0.2.0 since it adds new features and isn't a simple patch

Reply:
Fixed